core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -706,6 +703,7 @@ public PropServerConfiguration(
             this.telemetryEnabled = getBoolean(properties, env, "telemetry.enabled", true);
             this.telemetryDisableCompletely = getBoolean(properties, env, "telemetry.disable.completely", false);
             this.telemetryQueueCapacity = Numbers.ceilPow2(getInt(properties, env, "telemetry.queue.capacity", 512));
+            this.o3PartitionPurgeListCapacity = getInt(properties, env, "cairo.o3.partitionPurgeListInitialCapacity", 64);
```

Comment:
nit: other property names are dot-divided, so this one could be `cairo.o3.partition.purge.list.initial.capacity`

Reply:
done in https://github.com/questdb/questdb/pull/1778/commits/4cd8ecd9b8d0647988fda61943d386c0a2d5ab46